packages/host/src/Host.tsx

```diff
@@ -25,7 +25,14 @@ const Content = styled(Box)({
   overflowY: 'scroll',
   height: '100vh',
 });
-const queryClient = new QueryClient();
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+    },
+  },
```

Comment:
was annoying me that whenever I click dev tools and back on the app it refetches 🤔